m4_sessions/1806-inning/group7.md

```diff
@@ -1,33 +1,38 @@
 ## Group Members:
+Jim Dechant, Tony Robertson, Tyler Lundgren, Tyler McNamara
 
 
-
-## Name of Session: 
+## Name of Session: Lets just fuckin relax!
 
 ## Session Description 
 
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+ It's week 5, you are not stressed at all, but if you are this is the gear up for you. ( We will be stretching and meditating but isn't required )
 
 ## Session Outcomes 
 
-(What will students learn/leave with?): 
+This is mainly to take some time to yourself and relax, and hopefully you will have a few tools to take with you and use in times of stress. 
 
 ## Session Outline 
 
-(Please include who is facilitating and how much time you are going to spend in each section):
+Tyler L. => Everyone => Our Introductions - 5 min
+
+Jim D. => Icebreakers w/ students - 15 min
+(Have you practice mindfulness / meditation in the past? If so what have you done?)
+
+Walk to park - 5 min
+
+Tony => Everyone => Volunteer responses - 10 min
+(What has been the most stressful situation (maybe constant) and how did you handle it, our examples)
 
+Tyler L. & Tyler M. (carpal) && Tony & Jim (general) => Stretching - 10 min
+(Hand / Carpal Tunnel streches && general stretching)
 
+Tyler M. => Guided Meditation - 5-10 min
 
-     Best practices:  No section should be longer than 20 minutes
-     Create a session that YOU would want to attend
-     Share resources in your slack channel
-     All sessions NEED to have a closing activity, reflection, discussion
-    
-## Recommended Trajectory 
-## Introductions 
-## Intro to new material
-## Small Group Activity
-## Closing Discussion
+Back to campus - 5min
 
+Tyler M. => Strategies going forward - 10 min
+(How to incoporate a new habit, useful apps )
 
 Submit a PR at the end of this session and tag @emhickmann
+# Yo EllenMary we would like to have some stretching as part of our session, would it be possible to have our session in Clark? Can we bring our students outside for our session?
```

Comment:
For sure!  You can start in Clarke and then go outside.  It will likely be SUPER hot, so you might want to find a shaded area in the park/garden.  Consider finding a spot where students can still hear you (it's not so loud it is hard to meditate).